### ISSUE TITLE
Upgrade to electron 10(chrome 85)

### ISSUE THREAD

**Issue by manojVivek** [responsively-org/responsively-app#456]

**Comment by adi-mohan** [comment]
Hi! May I work on this issue?

**Comment by manojVivek** [comment]
Yes please @adi-mohan. That would be fantastic. I'm assigning this to you.  

Please connect on slack if you have any questions: https://responsively.app/join-slack

**Comment by rebzden-baracoda** [comment]
@adi-mohan @manojVivek hi :) how is the progress going with this one? Maybe I could try to help? There's already electron version 11 with support for M1. Please let me know, the app is awesome and I would love to help!!

**Comment by manojVivek** [comment]
@rebzden-baracoda That would be great, please give it a try and let us know if you are getting started.

**Comment by jjavierdguezas** [comment]
we are tackling this by rewriting the entire app. You can follow the progress on #724

**Comment by oliveirajhony** [comment]
**A concrete data point for this, from today.**

The published release (`v1.18.0`, Feb 2026) still declares `electron: ^27.0.4`, which bundles **Chromium 118**. `main` is already on Electron 44, so this is no longer about the upgrade work — it is that no release carries it yet.

What that costs in practice: **Chromium 118 has no X25519 in WebCrypto** (it landed in [Chrome 133](https://developer.chrome.com/release-notes/133)). Any app that keeps an X25519 keypair in the browser — end-to-end encrypted apps typically do — fails inside Responsively while working in a normal Chrome:

```
DOMException: Failed to execute 'importKey' on 'SubtleCrypto': Algorithm: Unrecognized name
```

Minimal repro, in the console of any viewport:

```js
await crypto.subtle.generateKey({ name: "X25519" }, true, ["deriveBits"])
```

It is worth noting *where* this lands. The app signs in fine, the server accepts the credentials, and the failure only appears afterwards, when the browser tries to open the account's key — so it reads as a broken login rather than as a missing browser feature. `isSecureContext` is `true`, `crypto.subtle` exists, IndexedDB works; only that one algorithm is absent. It is easy to spend a while debugging your own app before suspecting the viewport.

Nothing here is a request for new work, and the multi-viewport workflow is genuinely hard to replace — that is why the gap is worth reporting rather than working around. Is there anything blocking a release off the Electron 44 line, or a prerelease build for people who need it now? Happy to test one.

**Comment by manojVivek** [comment]
A beta release will go out for this today and the stable release subsequently.
